apps/marketplace-controller/app/src/components/NewOrder/Order.js

```diff
@@ -1,12 +1,15 @@
 import React, { useEffect, useCallback, useContext, useMemo, useRef, useState } from 'react'
 import styled from 'styled-components'
-import { useApi, useAppState } from '@aragon/api-react'
+import { useApi, useAppState, useConnectedAccount, useCurrentApp } from '@aragon/api-react'
 import { Button, DropDown, GU, Info, Text, TextInput, theme, unselectable, useTheme } from '@aragon/ui'
 import { MainViewContext } from '../../context'
 import Total from './Total'
 import Info_ from './Info'
 import ValidationError from '../ValidationError'
 import { toDecimals, formatBigNumber } from '../../utils/bn-utils'
+import miniMeTokenAbi from '../../abi/MiniMeToken.json'
+import marketMakerAbi from '../../abi/BancorMarketMaker.json'
+import ethers from 'ethers'
```

Comment:
Could we move this import above all local imports? Generally we do external library imports first.